### ISSUE TITLE
Pre-configured ssh keys for "cbsd node mode=add".

### ISSUE THREAD

**Issue by mzs114** [cbsd/cbsd#59]
Is it possible to use ssh keys which are already configured?

I guess when we add a new cbsd node, a new key is generated. Can we specify keys which are already configured on the remote node?

**Comment by olevole** [comment]
CBSD yet it does not have special ability for management public/private key. You can repeat follow steps from install_sshkey function by hand for replace generated key:

https://github.com/olevole/cbsd/blob/master/initenv.subr#L521

In short:

Private key must be placed in: $workdir/.ssh/${md5_fromname}.id_rsa"
Public key: $workdir/.ssh/authorized_keys"

where ${md5_fromname} - is md5sum from nodename:
~~~
% md5 -qs `cat ~cbsd/nodename`
~~~

**Comment by olevole** [comment]
upd: I've just leave the ticket in open state: todo "think about it"

**Comment by olevole** [comment]
note to self: we need to move authkey data into SQLite3 and wrote -tui/management cli for this. 
It planned to CBSD 11.0.6

**Comment by olevole** [comment]
Added (will be available with CBSD 11.0.10)  script:
~~~
cbsd sshkey [private_key=<path to personal id_rsa>] [public_key=<path to personal pub/authorized_keys>] [renew=1]
~~~

to control node ssh/pubkey pair. 

Sorry about the delay ;-)
